### DISCUSSION TITLE
Get Ready For Universe 🎉

### DISCUSSION THREAD

**Discussion by samus-aran** [community/events#17] [Announcements]
![GitHub Universe 2022](https://user-images.githubusercontent.com/4041275/200894926-c6186fba-d697-4c83-978b-ee3353d047cf.png)

Get ready for Universe with a fun, behind-the-scenes tour of YBCA. Meet some of the speakers and special guests, and get an exclusive preview of what’s to come before CEO Thomas Dhomke takes the stage for the Universe keynote. 🎉

**Comment by mrdiamonddirt** [comment]
i think i'm ready 🥰

**Reply by ossanna16** [reply]
Yay, so are we!

**Reply by hrthealer** [reply]
What a fantastic performance

**Reply by Patecatl848** [reply]
Oh yes, 🤓👨🏻‍💻🧪🧠🦾🩻🐍⚕️

**Comment by nul800sebastiaan** [comment]
Any way the live streaming peeps can boost the volume? I'm at 100% and it's still very low.. 😅

**Reply by mrdiamonddirt** [reply]
sounds fine for me 😃 you not got a headset 🎧

**Reply by nul800sebastiaan** [reply]
Maxed out on noise cancelling headset!

**Reply by hrthealer** [reply]
Sound is GR8 for me

**Reply by hrthealer** [reply]
I am having the time of my life, love the presentations as well as the experience.  I will now want to look into a more professional membership for the additional services.

**Comment by Moulick** [comment]
The stream on youtube is a good 10 seconds ahead of the stream on the GitHub universe website
https://www.youtube.com/watch?v=wKZg0DScucw

**Comment by AccessiT3ch** [comment]
Accessibility of the schedule page needs some love...tabbing through is gnarly, glad I'm not using a screen reader for it!

**Reply by erickolivares** [reply]
Yeah, I was a bit surprised about he UX decisions compared to years prior.

**Reply by ossanna16** [reply]
Thank you for the feedback @AccessiT3ch!

**Reply by AccessiT3ch** [reply]
Where can we open issues?

<img width="349" alt="Screenshot 2022-11-09 at 1 05 40 PM" src="https://user-images.githubusercontent.com/95449364/200942222-f5097929-1228-470f-aefc-fdcc5cc669f2.png">

**Reply by ossanna16** [reply]
Hi there @AccessiT3ch, what would you like to open an issue for?

**Reply by AccessiT3ch** [reply]
Well in this case, the color contrast of the session lineup link on the main schedule page. 😄

**Reply by ossanna16** [reply]
@AccessiT3ch Please create a Discussion here and we will make sure to label it as an Accessibility issue and pass on the feedback!

**Reply by AccessiT3ch** [reply]
Created! https://github.com/community/events/discussions/53 👀

**Comment by Jaquinor** [comment]
Is simultaneous translation available?

**Reply by samus-aran** [reply]
Thank you for highlighting this. We do support captions English, French, Spanish & Portuguese via the CC button on the GitHubUniverse streaming. 👍 

<img width="444" alt="Screenshot 2022-11-09 at 19 47 25" src="https://user-images.githubusercontent.com/4041275/200927044-e48376ad-bf46-42f9-a298-d2600f4d3c28.png">

**Comment by mishmanners** [comment]
READY

**Comment by Patecatl848** [comment]
I want to say how grateful I am to all those that have contributed to this event, I have heard and absorbed so much information from Codespace, Codex AI, Cloud CI/CD, Security and so much more. Thank you so much for supporting me and all developers.

**Comment by henriavo** [comment]
are there universe '22 wallpapers somewhere?? love the colors.

**Reply by ossanna16** [reply]
Hi there @henriavo! No wallpapers sadly but we have other Universe items in the [GitHub Store](https://thegithubshop.com/collections/universe-2022-collection).
